amoro-ams/amoro-ams-metrics/amoro-metrics-prometheus/src/main/java/org/apache/amoro/metrics/promethues/PrometheusMetricsReporter.java

```diff
@@ -52,9 +52,10 @@ public void close() {
     this.server.close();
   }
 
+  /** {@inheritDoc} */
   @Override
   public String name() {
-    return "prometheus-exporter";
+    return "prometheus";
```

Comment:
`prometheus ` seems to be ambiguous. I am not sure if the service name must be the same with the plugin name.

Reply:
Yes it is. AMS need to load the plugin class base on the name, and plugin name is same as property name in configmap

Reply:
When I first used the -exporter suffix considering because that we might implement a reporter based on push-gateway in the future.

Reply:
What's problem for '-' in kubernetes ?

Reply:
'-' is valid, the problem is the name should be less than 15 characters

Reply:
'-' is valid +1 ,can use other replace it

Reply:
Do we have to use SVC to expose exporter in K8S?
What are the common ways to integrate Promethues in K8S?

Reply:
prometheus push-gateway has limitations for some scenarios and not fitable for long-lived jobs(refer to https://prometheus.io/docs/practices/pushing/#should-i-be-using-the-pushgateway). I suggest we use prometheus  operator + podmonitor to achive this

Reply:
How about `prom-exporter`? exporter seems to be an important part of the plugin name.

Reply:
@zhoujinsong I think it's fine, as long as it's stated that it's an abbreviation of prometheus

Reply:
@baiyangtx How about adding a additional property `portName` which can be used as `containerPort` in deployment and `port` in podmonitor
```
plugin:
  metricReporters:
    - name: prometheus-exporter
      enabled: true
      properties:
        port: 7001
        portName: metrics # which is hereby normatively defined as https://datatracker.ietf.org/doc/html/rfc6335#section-5.1
        interval: 60s
      service:
        type: ClusterIP
        port: "7001"
        nodePort: ~
        annotations: {
          "prometheus.io/scrape": "true",
          "prometheus.io/path": "/metrics",
          "prometheus.io/port": "7001"
        }
```

Reply:
```
apiVersion: monitoring.coreos.com/v1
kind: PodMonitor
metadata:
  labels:
    app.kubernetes.io/name: amoro-monitor
  name: amoro-monitor
spec:
  namespaceSelector:
    any: true
  podMetricsEndpoints:
    {{- range .Values.plugin.metricReporters }}
    - interval: {{ .properties.interval }}
      port: {{ .properties.portName }}
    {{- end}}
  selector:
    matchLabels:
      app.kubernetes.io/name: ams
```